src/style.css

```diff
@@ -61,7 +61,7 @@ body {
   background-size: 11px 11px;
 }
 
-.search-box--small input[type="search"]:focus {
+.search-box--small .search-box-field {
```

Comment:
Try to avoid increasing the specificity of the selector. One class is enough to identify the elements on the page. No need to write something like .block.colored-block. Use .block and .block--colored instead, and apply each of them to your element

Reply:
я наверное тупой(